modules/league_stats.rb

```diff
@@ -5,83 +5,43 @@ def count_of_teams
   end
 
   def best_offense
-    total_goals_by_team = {}
```

Comment:
This really DRY's up all the methods in LeagueStats module.

Reply:
Much drier.